<launch>
	<node pkg="rex_gridmap_buffer" type="rex_gridmap_buffer" name="rex_gridmap_buffer" output="screen">
    <param name="input_topic" value="/rex_gridmapping/elevation_map_raw"/>
	</node>

  <!--Adding in the default transform lookups so they don't give errors! Remove these if provided elsewhere-->
  <node pkg="tf2_ros" type="static_transform_publisher" name="tf2_static_map_odom"
    args="0 0 0 0 0 0 1 map odom" />
  <node pkg="tf2_ros" type="static_transform_publisher" name="tf2_static_odom_base_link"
    args="0 0 0 0 0 0 1 odom base_link" />
  
</launch>
